### DISCUSSION TITLE
FDS cannot determine which boundary cell to assign.

### DISCUSSION THREAD

**Discussion by hunzideriji** [firemodels/fds#13442] [Q&A] [answered]
Please help me take a look at my code. I have changed the coordinates of Net Heat Flux -INSTANT VALUE, but it still does not solve the error problem
[Uploading 新建文本文档.txt…]()

**Comment by hunzideriji** [comment]
[新建文本文档.txt](https://github.com/user-attachments/files/17042729/default.txt)

**Reply by drjfloyd** [reply]
Set IOR.

**Reply by hunzideriji** [reply]
![微信图片_20240918200438](https://github.com/user-attachments/assets/b3d8e405-8c9a-4a8c-8a26-da46bcd3b653)
Excuse me, is this how it is set up， But there will still be errors

**Reply by drjfloyd** [reply, marked as answer]
NET HEAT FLUX is a solid phase quantity. Make sure you are placing it close enough to the solid surface where you want to measure the flux for FDS to find the surface.

**Reply by hunzideriji** [reply]
THANKS
